releases/patch-releases.md

```diff
@@ -93,14 +127,40 @@ End of Life for **1.19** is **2021-09-30**
 | 1.19.2        | 2020-09-11           | 2020-09-16  |
 | 1.19.1        | 2020-09-04           | 2020-09-09  |
 
+### 1.18
+
+**1.18** enters maintenance mode on **2021-04-28**
+
+End of Life for **1.18** is **2021-04-30**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.18.19       | 2021-04-26           | 2021-04-28  |
```

Comment:
I would like to propose final patch release for 1.18 on 5/7/21 with the rest of the active branch patches. This is simpler for folks trying to get cherry picks backported to all branches, and also provides more time given that we are a bit late communicating this date.

Reply:
@hasheddan -- I'd like to minimize how much longer we keep the 1.18 branch around, as it's running on go1.13.15 (which is out of support upstream).

The thought process around the 28th was although it deviates from the patch release schedule month-to-month, it allows us to give a very consistent response about when a release in entering maintenance mode or going EOL; it's _always_ the 28th of the designated month.

A consideration for a follow-up 👇🏾 
Are we okay with 5/7 as a patch release date next month? That falls within the week of KubeCon.

Reply:
That sounds like a good policy (having final patch be on the 28th of the EOL month), but I would encourage us to consider enacting that post 1.18 as we are already close to the deadline and some folks have made the assumption (reasonably) that if we were to have one more patch it would be aligned with the other branches. We should also consider any future confusion this could introduce with folks opening CPs to branches that have different CP deadlines when the oldest active branch is reaching EOL.

Reply:
> some folks have made the assumption (reasonably) that if we were to have one more patch it would be aligned with the other branches

@hasheddan -- Great point. Updated!

> Are we okay with 5/7 as a patch release date next month? That falls within the week of KubeCon.

Don't mind me. Looking again, I see that this is the cherry pick deadline, with the patches to happen the week after (which should be fine).